client/src/Components/FeelingsLog/FeelingsLog.js

```diff
@@ -39,9 +39,21 @@ const FeelingsLog = ({logs}) => {
         {userLogs}
         </tbody>
       </table>
-
     </section>
   )
 }
 
-export default FeelingsLog;
\ No newline at end of file
+export default FeelingsLog;
+
+FeelingsLog.propTypes = {
+  logs: PropTypes.arrayOf(PropTypes.shape({
+    entry_id: PropTypes.number.isRequired,
+    user_id: PropTypes.string,
+    date: PropTypes.string.isRequired,
+    feeling: PropTypes.string.isRequired,
+    journal_entry: PropTypes.string.isRequired,
+    created_at: PropTypes.string,
+    updated_at: PropTypes.string,
+    activity: PropTypes.string.isRequired
```

Comment:
Another awesome use of `.shape()`!! I love this formatting, too - it makes it really easy to look at this and understand exactly what you're expecting to be passed in. Awesome work!!